### PR TITLE
Chore/rollup babel

### PR DESCRIPTION
**Pull request by zero0Halo**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [ ] Tested my solution on Mobile & Tablet.
- [ ] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [ ] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [ ] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [ ] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.

---------
**Outline your feature or bug-fix below**

Our exports now all run through babel upon running `build`.  This fixes a bug with IE11, which couldn't use Anchor because its lack of support of many standard features.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -15,6 +16,10 @@ const commonPlugins = [
     }),
     sourceMaps(),
     terser(),
```
Comment (sharkparty) [inline rollup.config.js]: Can you comment out terser? Separate from your work, I wanna see if this is impacting intellisense.

File: rollup.config.js [hunk 2]
```diff
@@ -119,6 +124,7 @@ const componentList = {
     
     
 export default {
+    format: "iife",
```
Comment (sharkparty) [inline rollup.config.js]: Why an IIFE here?

**Review by sharkparty — CHANGES_REQUESTED** [review]
Changes requested on something completely unrelated to this PR. Sorry.

File: rollup.config.js (hunk 2 above)
Comment (zero0Halo) [reply rollup.config.js]: Was following multiple guides/docs on getting babel working. I can remove this and see if it causes a problem.

Comment (sharkparty) [reply rollup.config.js]: I’m curious why to transpire this down into an IIFE and how that impacts chunking. You should be able to see the output if you run the build script locally.

**Review by sharkparty — COMMENTED** [review]
Still need to know what this is/was doing.

**Review by sharkparty — APPROVED** [review]
LGTm

**Comment by sharkparty** [comment]
We should be able to remove the TSC script since Babel does that for us.

**Review by bjohnsonrmn — COMMENTED** [review]
This looks fine to me, but should this branch also update `yarn.lock` since it updates `package.json`?

**Comment by zero0Halo** [comment]
> This looks fine to me, but should this branch also update `yarn.lock` since it updates `package.json`?

I'm not sure. I have a tendency to use npm and not yarn, tough habit to break.
